antlr4-testgen-maven-plugin/pom.xml

```diff
@@ -36,9 +36,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
     <modelVersion>4.0.0</modelVersion>
-
-    <groupId>org.antlr</groupId>
-    <version>4.5.1-SNAPSHOT</version>
+    <parent>
```

Comment:
:bulb: This change makes the `<project.build.sourceEncoding>` property below unnecessary.